heron/healthmgr/src/java/org/apache/heron/healthmgr/HealthPolicyConfigReader.java

```diff
@@ -49,6 +49,7 @@
   public enum PolicyConfigKey {
     CONF_FILE_NAME("healthmgr.yaml"),
     HEALTH_POLICIES("heron.class.health.policies"),
+    HEALTH_POLICY_MODE("health.policy.mode"),
     HEALTH_POLICY_CLASS("health.policy.class"),
     HEALTH_POLICY_INTERVAL("health.policy.interval.ms"),
```

Comment:
add the `_MS` suffix to `HEALTH_POLOGY_INTERVAL`

Reply:
updated